lte/gateway/python/magma/subscriberdb/main.py

```diff
@@ -55,11 +56,20 @@ def main():
 
     # Start a background thread to stream updates from the cloud
     if service.config['enable_streaming']:
-        callback = SubscriberDBStreamerCallback(store, service.loop)
-        stream = StreamerClient({"subscriberdb": callback}, service.loop)
-        stream.start()
+        grpc_client_manager = GRPCClientManager(
+            service_name="subscriberdb",
+            service_stub=SubscriberDBCloudStub,
+            max_client_reuse=60,
```

Comment:
If it prevents bugs, we can set it to 0, if we are  getting subscriber state once in 3 minutes getting a new connection doesn't sound expensive.